python/lsst/skymap/tractBuilder.py

```diff
@@ -102,7 +102,8 @@ def getPatchInfo(self, index, tractWcs):
 
         Parameters
         ----------
-        index : `lsst.skymap.Index2D` or `Iterable` [`int`, `int`]
+        index : `lsst.skymap.Index2D` or `~collections.abc.Iterable` \
+                [`int`, `int`]
```

Comment:
I guess this means "iterable returning two integers", but `Iterable[int, int]` is probably an incorrect syntax for mypy?

Reply:
Yes. I think it might actually mean iterable of two integers because `Index2D` is a single x,y coordinate.  I did ponder it for a few seconds and since I was unsure I left it as it was before.